Sources/Bot/Services/MergeService.swift

```diff
@@ -132,14 +153,20 @@ extension MergeService {
             self.status = status
         }
 
-        static func initial(targetBranch: String, integrationLabel: PullRequest.Label, topPriorityLabels: [PullRequest.Label], statusChecksTimeout: TimeInterval) -> State {
+        static func initial(
+            targetBranch: String,
+            integrationLabel: PullRequest.Label,
+            topPriorityLabels: [PullRequest.Label],
+            statusChecksTimeout: TimeInterval,
+            isWaitingForInitialPullRequests: Bool
```

Comment:
[nit] I fill like naming this parameter as `isStarting` or `isIdle` will make it more clear